modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Deployment.java

```diff
@@ -19,7 +20,7 @@
  * Defines the Deployment.
  */
 @Data
-public class Deployment {
+public class Deployment implements Serializable {
 
```

Comment:
Add serialVersionUID to avoid potential incompatibilities during deserialization.